web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -78,7 +78,7 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
     showNoExtractionTab
 }) => {
     const getMetadata = (state: JobValues) => {
```

Comment:
One suggestion of improvement [optional] would be to move this function to a separate file.

Reply:
not sure if this is necessary here